### PR TITLE
Update free-programming-books-subjects.md

### PR DESCRIPTION
**Pull request by AryanParashar24**
Added three Container books to the Containers Section

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by AryanParashar24** [comment]
Added three books links to Three Container books Please check it out

**Review by sohansai — COMMENTED** [review]
Access to the resources is not free.

Include freely available resources. Refer to docs/CONTRIBUTING.md for contribution guidelines.

**Comment by AryanParashar24** [comment]
@sohansai sry for the previous inclusions plz have a look on the updated code

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -246,12 +246,15 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
 
 * [CI/CD for Monorepos: Effectively building, testing, and deploying code with monorepos](https://github.com/semaphoreci/book-monorepo-cicd) - Pablo Tomas Fernandez Zavalia, Marko Anastasov, SemaphoreCI (PDF, EPUB, Kindle)
 * [CI/CD with Docker and Kubernetes Book](https://github.com/semaphoreci/book-cicd-docker-kubernetes) - Marko Anastasov, Jérôme Petazzoni, Pablo Tom F. Zavalia, SemaphoreCI (PDF, EPUB, Kindle)
+* [Container Networking Docker Kubernetes]([https://www.oreilly.com/library/view/container-networking/9781492036845/](https://pdfcoffee.com/qdownload/container-networking-docker-kubernetes-pdf-free.html)) - Michael Housenblas (HTML)
```
Comment (sohansai) [inline books/free-programming-books-subjects.md:249]: Remove this. We do not support PDFDrive links. We only allow links from authorized sources

File: books/free-programming-books-subjects.md [hunk 2]
```diff
@@ -246,12 +246,15 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
...
 * [Docker Jumpstart](https://odewahn.github.io/docker-jumpstart/) - Andrew Odewahn
 * [Docker Tutorial](https://people.irisa.fr/Anthony.Baire/docker-tutorial.pdf) - Anthony Baire (PDF)
 * [Docker Tutorial](https://www.tutorialspoint.com/docker/) - Tutorials Point (HTML, PDF)
 * [Dotnet Microservices Architecture for Containerized NET Applications](https://aka.ms/microservicesebook) - Cesar de la Torre, Bill Wagner, Mike Rousos (PDF)
 * [Kubernetes Deployment & Security Patterns](https://resources.linuxfoundation.org/LF+Projects/CNCF/TheNewStack_Book2_KubernetesDeploymentAndSecurityPatterns.pdf) - Alex Williams (PDF)
 * [Kubernetes for Full-Stack Developers](https://www.digitalocean.com/community/books/digitalocean-ebook-kubernetes-for-full-stack-developers) - Jamon Camisso, Hanif Jetha, Katherine Juell (PDF, EPUB)
+* [Kubernetes: Up and Running](https://eddiejackson.net/azure/Kubernetes_book.pdf) - Kelsey Hightower, Brendan Berns, Joe Beda (PDF)
```
Comment (sohansai) [inline books/free-programming-books-subjects.md:256]: This is not an authorized distribution. Refer to docs/CONTRIBUTING.md for guidelines.

File: books/free-programming-books-subjects.md [hunk 3]
```diff
@@ -246,12 +246,15 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
...
+* [The Complete Guide to Kubernetes Security](https://start.paloaltonetworks.com/complete-guide-kubernetes?sfdcid=7014u000001Z9gvAAC&gad_source=1)
```
Comment (sohansai) [inline books/free-programming-books-subjects.md:257]: Please share only free content. Ensure it's completely free. We cannot include links to pages needing working email addresses for book access. Although we typically don't list books requiring registration, certain course platforms may need an email account for proper functionality.

**Pull request closed without merging**